pkg/booleanpolicy/violationmessages/printer/util.go

```diff
@@ -80,7 +80,7 @@ func getSingleValueFromFieldMap(f string, fieldMap map[string][]string) (string,
 	return values[0], nil
 }
 
-func executeTemplate(tpl string, values interface{}) ([]string, error) {
+func getTemplate(tpl string) (*template.Template, error) {
```

Comment:
Why can't we use `executeTemplate` instead of `executeTemplateT`? 

From the performance perspective amount of calls to `template.New("").Parse(tpl)` is the same considering the fact that we use lazy cache here

Reply:
Mini-optimization: `executeTemplate` makes a number of extra steps like searching for the template and returning a single value array. All this looks unnecessary to do in a loop.

Reply:
That's true. However, if we keep using `executeTemplate` it will not only keep this `util` file more simple and thus more convenient for future use, but also simplify `imageSignatureVerifierPrinter`.

From what I saw while looking at CPU and memory dumps some time ago, we really don't spend much CPU or memory to generate violations. Thus difference made by this optimization will probably not affect us in any way. 

So my suggestion is to use `executeTemplate`

Reply:
Ok!